src/server/game/Conditions/ConditionMgr.h

```diff
@@ -126,8 +126,8 @@ enum ConditionSourceType
     CONDITION_SOURCE_TYPE_CREATURE_TEMPLATE_VEHICLE      = 16,
     CONDITION_SOURCE_TYPE_SPELL                          = 17,
     CONDITION_SOURCE_TYPE_SPELL_CLICK_EVENT              = 18,
-    CONDITION_SOURCE_TYPE_QUEST_ACCEPT                   = 19,
-    CONDITION_SOURCE_TYPE_QUEST_SHOW_MARK                = 20,
+    CONDITION_SOURCE_TYPE_QUEST_AVAILABLE                = 19,
+    // Condition SourceType 20 unused                    = 20,
```

Comment:
shouldn't we leave a placeholder here and output an error if someone tries to use the `20` ?